Framework/Core/src/runDataProcessing.cxx

```diff
@@ -1951,6 +1951,71 @@ void overridePipeline(ConfigContext& ctx, WorkflowSpec& workflow)
   }
 }
 
+void overrideLabels(ConfigContext& ctx, WorkflowSpec& workflow)
+{
+  struct LabelsSpec {
+    std::string_view matcher;
+    std::vector<std::string_view> labels;
```

Comment:
Why a vector of `string_views` and not of `strings`?

Reply:
I hoped for less string copies during `vector` (re)allocation, both in `LabelsSpec.labels` and `std::vector<LabelsSpec>`.

Reply:
sure, but string_view does not have any ownership on the data, where is it actually stored?

Reply:
Here is the original string:
```
auto labelsString = ctx.options().get<std::string>("labels")
```
It lives long enough before the labels are actually copied into `DataProcessorSpec::labels`.

Though I must admit I also wanted to exercise `string_view` a bit, so this may look like excessive optimization.

Reply:
I would avoid it, yes. std::string is probably fine.

Reply:
sure, i'll change it.